KWYG-28 Splash screen anzeigen

Floskinner opened the pull request:
## Änderungen
Die PWA unterstützt nun auch Splash Screens auf iOS und Androide. Zudem wurden weitere Attribute zur `manifest.json` hinzugefügt für bessere PWA Kompatibilität.
-> Closes #43 

## Tests mit Google Chrome Lighthouse
![image](https://user-images.githubusercontent.com/58706771/209715165-99e00489-ccc9-4dcf-bceb-7f112565ad5c.png)

Floskinner committed:
KWYG-30 Add CNAME file for gh pages

Floskinner committed:
KWYG-30 Only trigger if a tag event is fired

johannesbrandenburger committed:
WikiInfo (KWYG-11)

johannesbrandenburger committed:
improved code readability

johannesbrandenburger committed:
MemoFetcher

johannesbrandenburger committed:
disabled fulltext wiki search

johannesbrandenburger committed:
wiki link

Floskinner committed:
added additional infos for a good pwa

Floskinner committed:
changed current icons to own icons and added icons for iOS and Androide PWA

johannesbrandenburger committed:
Merge pull request #41 from DHBW-FN-TIT20/KWYG-11-Informationen-von-Wikipedia-anzeigen

Kwyg 11 informationen von wikipedia anzeigen (css kommt später)

johannesbrandenburger committed:
routing

johannesbrandenburger committed:
toggleTileLayer (satellite)

johannesbrandenburger committed:
remove unnecessary import tilelayer

johannesbrandenburger committed:
Merge pull request #40 from DHBW-FN-TIT20/KWYG-30-Automatisches-Deployen-verbessern

KWYG-30 Automatisches deployen verbessern

johannesbrandenburger commented:
@Floskinner Wenn man ein weißes Browser-Theme nutzt, sieht man kein Logo.
<img width="548" alt="Screenshot 2023-01-01 at 16 36 41" src="https://user-images.githubusercontent.com/79154528/210176442-7fbc7283-3918-415c-a3fb-92d1ba1df9e3.png">

johannesbrandenburger committed:
photon.komoot search autocompletion

johannesbrandenburger committed:
german map titles

johannesbrandenburger committed:
Fix first location state change wiki not loading

johannesbrandenburger committed:
Merge pull request #45 from DHBW-FN-TIT20/KWYG-12-Fahrroute-auf-ausgewählte-Position-auf-der-Karte-anzeigen

Kwyg 12 Fahrroute auf ausgewählte Position auf der Karte anzeigen + Satellitenansicht

johannesbrandenburger committed:
Merge pull request #46 from DHBW-FN-TIT20/photon.komoot-search-autocompletion

Photon.komoot search autocompletion

Floskinner commented:
> @Floskinner Wenn man ein weißes Browser-Theme nutzt, sieht man kein Logo. <img alt="Screenshot 2023-01-01 at 16 36 41" width="548" src="https://user-images.githubusercontent.com/79154528/210176442-7fbc7283-3918-415c-a3fb-92d1ba1df9e3.png">

Überlegung ist hier vom CSS Team, einen `#202020` runden Hintergrund hinzuzufügen, damit man das Logo auch immer sieht.

@screetox  oder @baldur132 könnte das einer machen bzw. das Logo anpassen / ihr euch absprechen?

Commit:
updated design files

updated icons with background

updated iOS splash screens

removed old splash screens

changed default color to `#222222`

Author:    baldur <baldur@siegels.us>
Date:      Tue Jan 3 16:46:03 2023 -0700

eira132 commented:
@Floskinner ich habe die Icons alle mit einem Rundeck versehen, jedoch mit der Farbe `#222222`, da das mit dem Mockups übereinstimmt. 
![image](https://user-images.githubusercontent.com/43816694/210460880-d069eddf-4eff-4fdd-850b-fceff2565021.png)

Zusätzlich habe ich alle die iOS Splash Screens geupdatet, da die den falschen Font hatten.
Die Icons und Splash Screens sind jetzt alle (designtechnisch) zusammen verwaltet in `design/kwyg-logo-exports.svg`, wo die auch entsprechend als Batch exportiert werden können (zumindest mit Inkscape).
![image](https://user-images.githubusercontent.com/43816694/210461134-22b6fca2-137f-46c1-9e2f-339828018012.png)

Floskinner committed:
Merge branch 'KWYG-Sprint-2' into KWYG-28-splashscreen-anzeigen

johannesbrandenburger commented:
@Floskinner hast du dir die Changes von Baldur nochmal durchgeschaut? Dann könnt man nämlich mergen 👍.

Floskinner commented:
@johannesbrandenburger noch nicht mergen. Anscheinend ist die manifest.json fehlerhaft geworden... idk why
Zudem wollte ich nochmal testen ob alles geht

Floskinner commented:
> @johannesbrandenburger noch nicht mergen. Anscheinend ist die manifest.json fehlerhaft geworden... idk why Zudem wollte ich nochmal testen ob alles geht

..okay geht alles. War wohl wieder so ein netter Service Worker Cache Fehler
Kann man nun mergen
